onnxruntime/test/providers/cpu/tensor/squeeze_op_test.cc

```diff
@@ -134,6 +140,11 @@ TEST(SqueezeOpTest, Squeeze_2_axes_input) {
 }
 
 TEST(SqueezeOpTest, Squeeze_Empty_Axes_opset13) {
+  // TODO: Unskip when fixed #41968513
+  if (DefaultDmlExecutionProvider().get() != nullptr) {
+    GTEST_SKIP() << "Skipping because of the following error: MLOperatorAuthorImpl.cpp(2068): Catastrophic failure";
```

Comment:
Aah, we have a test case for empty axes, but not one for completely missing attribute -_-.

```
    // float32. Remove all 1's.
    {
      "graph_name": "Squeeze version 13 remove all 1s",
      "op_type": "Squeeze",
      "version": 13,
      "axes": [],
      "data": {"dims": [1, 2, 1, 3], "values": [1, 2, 3, 4, 5, 6]},
      "squeezed": {"dims": [2, 3], "values": [1, 2, 3, 4, 5, 6]},
      "T": "float32"
    },
```